pallets/parachain-staking/src/test_staking_pot.rs

```diff
@@ -16,18 +31,21 @@ fn get_total_balance_of_collators(collator_account_ids: &Vec<AccountId>) -> u128
 
 #[test]
 fn fee_is_added_to_pot() {
+    let collator_1 = collator_1();
+    let collator_3 = collator_3();
+
     ExtBuilder::default()
-        .with_balances(vec![(1, 20), (2, 40), (3, 20), (4, 20)])
-        .with_candidates(vec![(1, 20), (3, 20), (4, 20)])
+        .with_balances(vec![(collator_1, 20), (non_collator_account_id(), 40), (collator_3, 20), (collator_4(), 20)])
```

Comment:
Any reason why some collators are immediate getters whilst some are set up front?

Reply:
Its based on usage. If an account is used in a lot of places, I use a variable. Otherwise I call the getter directly.